libs/application/types/src/lib/ApplicationTypes.ts

```diff
@@ -253,11 +253,11 @@ export const ApplicationConfigurations = {
   },
   [ApplicationTypes.OLD_AGE_PENSION]: {
     slug: 'ellilifeyrir',
-    translation: 'oap.application',
+    translation: ['oap.application', 'sia.application', 'uiForms.application'],
   },
   [ApplicationTypes.HOUSEHOLD_SUPPLEMENT]: {
     slug: 'heimilisuppbot',
-    translation: 'hs.application',
+    translation: ['hs.application', 'sia.application', 'uiForms.application'],
```

Comment:
Why is this change needed? 🤔

Reply:
so the messages that we are using from these namespaces are translated. 
I did this PR so this would work: https://github.com/island-is/island.is/pull/13181

Reply:
Ah ok, thanks 😄